modules/ROOT/nav.adoc

```diff
@@ -18,15 +18,15 @@
  ** xref:import-api-specification-exchange.adoc[From Exchange]
  ** xref:import-api-specification-maven.adoc[From Maven]
  ** xref:import-api-specification-local-file.adoc[From a Local File]
- ** xref:import-api-specification-design-center.adoc[From Design Center]
+ ** xref:import-api-specification-design-center.adoc[From MuleSoft VCS]
```

Comment:
Is VCS an acronym for the product? If so, is it legally branded?

Reply:
See my comment in the thread. Thanks!